war/src/main/scss/modules/dialogs.scss

```diff
@@ -0,0 +1,83 @@
+$jenkins-dialog-padding: 1rem;
+
+.jenkins-dialog {
+  border-radius: 0.6rem;
+  border: none;
+  background: transparent;
+  box-shadow: var(--modal-box-shadow);
+  backdrop-filter: var(--modal-backdrop-filter);
+  animation: jenkins-dialog-animate-in 0.25s cubic-bezier(0, 0.68, 0.5, 1.5);
+  overflow: hidden;
+  padding: 0;
+
+  &::backdrop {
+    background: var(--modal-backdrop-background);
+    animation: jenkins-dialog-backdrop-animate-in 0.15s;
+  }
+
+  &__contents {
+    overflow-y: auto;
+    overflow-wrap: break-word;
+    padding: $jenkins-dialog-padding;
+    max-height: 75vh;
+
+    &--title {
+      padding-top: 0;
+    }
+  }
+
+  &__input {
+    display: flex;
+    padding: 0 1rem 1rem;
+  }
+
+  &__buttons {
+    padding: 0.5rem 1rem 1rem;
+    justify-content: center;
```

Comment:
Buttons are centered in the dialogs. With OK button to the left. Is this ok or should they be put to the left or right side of the dialog and/or have the OK button on the right?